scripts/coverage

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e
```

Comment:
Again, good idea to make the script exit directly after an error! This helps ensuring that the script doesn't continue executing in an unexpected state.